### PR TITLE
Fixes for PIPs

### PR DESCRIPTION
**Pull request by adamdossa**
- move beneficiaries to proposals (from metadata)
- fail a PIP if there are not sufficient funds for a disbursal
- allow cool off period to be changed via governance
- check cool off period < proposal duration

### CODE REVIEW

File: pallets/pips/src/lib.rs [hunk 1]
```diff
@@ -888,6 +924,7 @@ decl_module! {
                 Zero::zero(),
                 Zero::zero(),
                 proposal_data,
+                //beneficiaries,
```
Comment (fmiguelgarcia) [inline pallets/pips/src/lib.rs:927]: Pls  remove this.

File: pallets/runtime/develop/src/constants.rs [hunk 2]
```diff
@@ -9,7 +9,7 @@ pub mod time {
     // mainnet
     // pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 4 * HOURS;
     // Testnet
-    pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 5 * MINUTES;
+    pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 1 * MINUTES;
```
Comment (fmiguelgarcia) [inline pallets/runtime/develop/src/constants.rs:12]: Do we want this in this PR? or It is just for testing?

File: pallets/runtime/develop/src/runtime.rs [hunk 3]
```diff
@@ -295,7 +295,7 @@ pallet_staking_reward_curve::build! {
 
 parameter_types! {
     // Six sessions in an era (24 hours).
-    pub const SessionsPerEra: sp_staking::SessionIndex = 6;
+    pub const SessionsPerEra: sp_staking::SessionIndex = 3;
```
Comment (fmiguelgarcia) [inline pallets/runtime/develop/src/runtime.rs:298]: Do we want this in this PR? or It is just for testing?

Comment (adamdossa) [reply pallets/runtime/develop/src/runtime.rs:298]: Yep - was intentional ;-)

**Pull request merged**